app/src/main/java/com/codepath/apps/critterfinder/models/PetSearch.java

```diff
@@ -0,0 +1,134 @@
+package com.codepath.apps.critterfinder.models;
```

Comment:
I'd move this into our services package instead of our model since it's implementing a search service.